ydb/docs/en/core/integrations/liquibase.md

```diff
@@ -0,0 +1,551 @@
+| `Int8`, `Int16`, `Int32`, `Int64`       | A signed integer                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| `Uint8`, `Uint16`, `Uint32`, `Uint64`   | An unsigned integer                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `Text`, `Bytes`, `Json`, `JsonDocument` | Represent as text                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `Float`, `Double`, `Decimal(22, 9)`     | A real number                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `Interval`                              | [Format](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html#parse(java.lang.CharSequence)) `java.time.Duration`, prefix `PT` is optional                                                                                                                                                                                                                                                                                       |
+| `Date`, `Datetime`, `Timestamp`         | Format [ISO-8601](https://ru.wikipedia.org/wiki/ISO_8601), in Java, the following format is used: [ISO_LOCAL_DATE](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE), [ISO_LOCAL_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME), [ISO_INSTANT](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT) |
```

Comment:
it's still not clear which of these formats you're supposed to put into a CSV file to be imported + in which precision (YDB's or Java's) as they are different + what happens with timezones